generate/src/contract/events.rs

```diff
@@ -300,14 +300,6 @@ fn expand_builder_type(event: &Event) -> Result<TokenStream> {
                 self
             }
 
-            /// Limit the number of events that can be retrieved by this filter.
-            ///
-            /// Note that this parameter is non-standard.
-            pub fn limit(mut self, value: usize) -> Self {
```

Comment:
Why did the limit go?

Reply:
I think that was removed in a separate PR - you must have reviewed between rebasing.

Reply:
I created #315 to add it back in.